### ISSUE TITLE
sys eeprom is not attached in the s6100 init script

### ISSUE THREAD

**Issue by lguohan** [Azure/sonic-platform-modules-dell#4]

**Comment by lguohan** [comment]
```
root@str-s6100-acs-1:/home/admin# decode-syseeprom
/usr/bin/decode-syseeprom : ERROR : cannot load module: /usr/share/sonic/device/x86_64-dell_s6100_c2538-r0/plugins/eeprom.py
```

**Comment by stcheng** [comment]
is it verified? 
i still see
```
root@str-s6100-acs-2:/home/admin# decode-syseeprom
/usr/bin/decode-syseeprom : ERROR : [Errno 2] No such file or directory: '/sys/class/i2c-adapter/i2c-2/2-0050/eeprom'
```

**Comment by padmanarayana** [comment]
Though https://github.com/Azure/sonic-platform-modules-dell/pull/10 is merged @ 5ab014c,
https://github.com/Azure/sonic-buildimage/tree/master/platform/broadcom is still pointing to https://github.com/Azure/sonic-platform-modules-dell/tree/6cff824.

I think the debian package version was not modified for this commit - but is there a dependency with the debian package version in this case ?

**Comment by stcheng** [comment]
could you please update the submodule in the sonic-buildimage repository to point to the correct commit?

**Comment by padmanarayana** [comment]
Raised https://github.com/Azure/sonic-buildimage/pull/909 for this.

**Comment by padmanarayana** [comment]
root@sonic:~# show ver
SONiC Software Version: SONiC.master.0-12ba083
Distribution: Debian 8.9
Kernel: 3.16.0-4-amd64
Build commit: 12ba083
Build date: Thu Aug 24 06:56:45 UTC 2017
Built by: padman@ubuntu-16

Docker images:
REPOSITORY                TAG                 IMAGE ID            SIZE
docker-syncd-brcm         latest              33818fb966c2        318.4 MB
docker-orchagent-brcm     latest              4676e1dd780f        258.5 MB
docker-lldp-sv2           latest              66030eb6d136        256.6 MB
docker-dhcp-relay         latest              b435bdf94a27        253.6 MB
docker-database           latest              f13e2f9aa79f        251.8 MB
docker-snmp-sv2           latest              735c3490763a        291.2 MB
docker-teamd              latest              9ddefbceb76f        255.5 MB
docker-platform-monitor   latest              72790bc6f43c        271.1 MB
docker-fpm-quagga         latest              03a716ce4421        262.1 MB

root@sonic:~# show platform summ
Platform: x86_64-dell_s6100_c2538-r0
HwSKU: Force10-S6100
ASIC: broadcom

root@sonic:~#
root@sonic:~# decode-syseeprom
TlvInfo Header:
   Id String:    TlvInfo
   Version:      1
   Total Length: 166
TLV Name             Code Len Value
-------------------- ---- --- -----
Product Name         0x21   8 S6100-ON
Serial Number        0x23  20 CN0WKFYN779315810001
Base MAC Address     0x24   6 4C:76:25:F4:F9:00
Manufacture Date     0x25  19 08/02/2015 05:23:06
Device Version       0x26   1 0
Label Revision       0x27   3 X01
MAC Addresses        0x2A   2 384
Manufacturer         0x2B   5 77931
Manufacture Country  0x2C   2 CN
Vendor Name          0x2D   4 DELL
Diag Version         0x2E   4 1.10
Service Tag          0x2F   7 1VDSG02
Vendor Extension     0xFD   7
Platform Name        0x28  26 x86_64-dell_s6100_c2538-r0
ONIE Version         0x29   8 3.15.1.0
Part Number          0x22   6 0H68CJ
CRC-32               0xFE   4 0x284EEAA0
(checksum valid)
eeprom.pyc:~#

**Comment by stcheng** [comment]
thanks. i'll verify and close this issue.

**Comment by lguohan** [comment]
addressed in #10
